pkg/reconciler/ksvc/service_account_credentials.go

```diff
@@ -28,11 +28,13 @@ import (
 
 type CredentialBuilder struct {
 	client client.Client
+	config *v1.ConfigMap
 }
 
-func NewCredentialBulder(client client.Client) *CredentialBuilder {
+func NewCredentialBulder(client client.Client, config *v1.ConfigMap) *CredentialBuilder {
```

Comment:
Won't this configmap get out of date or are we re-instantiating this credential builder on each call.

Reply:
ye it is re-instantiating on each call.